sites/public/lib/helpers.tsx

```diff
@@ -57,7 +58,10 @@ const getListingImageCardStatus = (listing: Listing): StatusBarType => {
     formattedDate = openDate.format("MMM. D, YYYY")
     content = t("listings.applicationOpenPeriod")
   } else {
-    if (listing.applicationDueDate) {
+    if (listing.status === ListingStatus.closed) {
```

Comment:
would love to have tests for this but we are setting up public site unit tests in another issue 👍